kubernetes/rancher/README.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes with Rancher
+Run Kubernetes with a [Rancher](https://rancher.com/) Docker container.
+
+## Table of Contents
+  - [Description](#description)
+  - [Description](#prerequisites)
+  - [Running with xOpera](#running-with-xopera)
+
+## Prerequisites
+Minimum requirements for running Kubernetes with Rancher are:
+
+* Operating Systems
+  * Ubuntu 16.04 (64-bit)
+  * Red Hat Enterprise Linux 7.5 (64-bit)
+  * RancherOS 1.4 (64-bit)
+* Hardware
+  * 4 GB of Memory
+* Software
+  * Docker v1.12.6, 1.13.1, 17.03.2
+
+To install Docker on your machine you can use our [Docker xOpera example](../docker).
+
+## Description
+The main functionality of this solution is to run a Kubernetes service in a Rancher Docker container on a target 
+machine. The deployed Kubernetes dashboard is the available on `localhost:80` and `localhost:443`. Additionally one 
+[hello-world](https://hub.docker.com/_/hello-world) Docker container is deployed in the Rancher network and it will 
+show up in the dashboard.
+
+There are 2 main Ansible playbooks separated into folders. Usage is explained in the table below:
+
+|    Role      |   Purpose    |
+|:-------------|:-------------|
+| **rancher** | Installs Kubernetes by running Rancher Docker container |
+| **hello-world** | Runs hello-world |
```

Comment:
I'd name this `rancher-hello-world`.

Reply:
It's already named `docker-hello-world` though, so a change is needed eiter way :)

Reply:
This was updated.